src/structures/tss.rs

```diff
@@ -6,7 +6,7 @@ use core::mem::size_of;
 /// In 64-bit mode the TSS holds information that is not
 /// directly related to the task-switch mechanism,
 /// but is used for finding kernel level stack
-/// if interrupts arrive while in kernel mode.
```

Comment:
Technically the comment is correct for both kernel and user mode. I'm guessing you are thinking of the privilege stack table which is relevant when an interrupt/exception occurs in user mode, but the TSS also contains the interrupt stack table which is also relevant in kernel mode. Can you please reword the comment?